src/main/java/nl/tudelft/ti2206/group9/gui/Style.java

```diff
@@ -77,34 +100,55 @@ private Style() { }
      * They are loaded into Phongmaterials.
      */
     public static void loadTextures() {
-        final String path = "nl/tudelft/ti2206/group9/gui/";
-
-        final Image brickTexture = new Image(path + "texture_brick.png");
+        final Image brickTexture = new Image(PATH + "texture_brick.png");
         BRICK.setDiffuseMap(brickTexture);
 
-        final Image mossTexture = new Image(path + "texture_moss.png");
+        final Image mossTexture = new Image(PATH + "texture_moss.png");
         MOSS.setDiffuseMap(mossTexture);
 
-        final Image crackTexture = new Image(path + "texture_crack.png");
+        final Image crackTexture = new Image(PATH + "texture_crack.png");
         CRACK.setDiffuseMap(crackTexture);
 
-        final Image floorTexture = new Image(path + "texture_cobblestone.png");
+        final Image floorTexture = new Image(PATH + "texture_cobblestone.png");
         FLOOR.setDiffuseMap(floorTexture);
 
-        final Image coinTexture = new Image(path + "texture_coin.png");
+        final Image coinTexture = new Image(PATH + "texture_coin.png");
         COIN.setDiffuseMap(coinTexture);
 
-        final Image woodTexture = new Image(path + "texture_wood.png");
+        final Image woodTexture = new Image(PATH + "texture_wood.png");
         WOOD.setDiffuseMap(woodTexture);
 
-        final Image pillarTexture = new Image(path + "texture_pillar.png");
+        final Image pillarTexture = new Image(PATH + "texture_pillar.png");
         PILLAR.setDiffuseMap(pillarTexture);
 
-        final Image fenceTexture = new Image(path + "texture_fence.png");
+        final Image fenceTexture = new Image(PATH + "texture_fence.png");
         FENCE.setDiffuseMap(fenceTexture);
+    }
 
-        final Image playerTexture = new Image(path + "texture_player.png");
-        PLAYER.setDiffuseMap(playerTexture);
+    /**
+     * Method to load a playerTexture.
+     * @param location name of the texture.
+     * @return The phongmaterial of this texture
+     */
+    public static PhongMaterial loadPlayerTexture(final String location) {
+        final Image playerTexture = new Image(
+                PATH + "texture_" + location + ".png");
+        final PhongMaterial material = new PhongMaterial();
+        material.setDiffuseMap(playerTexture);
+        return material;
+    }
+
+    /**
+     * Method that creates all the skins.
+     */
+    public static void loadSkins() {
+        final int price = 9999;
+        ironMan = new IronManSkin(price, "Iron Man", "iron_man");
+        captain = new CaptainSkin(price, "Captain", "captain");
+        andy = new AndySkin(price, "Andy", "andy");
+        noob = new NoobSkin(0, "Noob", "noob");
+        boy = new BoySkin(price, "B-man", "b");
```

Comment:
I think "b" is a bit short for a texture name. I would suggest renaming it to "boy" or "b_man".

<review,naming